### PR TITLE
Revise Versioning Structure v3.4.0➔v3.4 and current➔next

### PR DESCRIPTION
**Pull request by nate-double-u**
Moving files.

Updating: config.toml content/docs/next/_index.md content/docs/next/metrics.md content/docs/v3.4/_index.md layouts/index.redirects layouts/partials/docs/hero.html layouts/partials/docs/nav-panel.html layouts/partials/home/hero.html

Closes: https://github.com/etcd-io/website/issues/152

/cc @chalin

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy previews:
* https://deploy-preview-161--etcd.netlify.app/
* https://deploy-preview-161--etcd.netlify.app/current
* https://deploy-preview-161--etcd.netlify.app/latest
* https://deploy-preview-161--etcd.netlify.app/
* https://deploy-preview-161--etcd.netlify.app/docs/v3.4.0/
* https://deploy-preview-161--etcd.netlify.app/docs/next

**Comment by nate-double-u** [comment]
I wonder if we should move what was `current` into `v3.4` now. Current/next has the updated descriptions that were introduced when we migrated the docs over from etcd-io/etcd, and I think current/next represents the latest `v3.4` version.

@ptabor @spzala @jberkus @chalin

**Comment by nate-double-u** [comment]
- [ ] I need to update the deprecation warning.

File: layouts/index.redirects [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {{ $latest := site.Params.versions.latest }}
 /docs/latest     /docs/{{ $latest }}/
 /docs/latest/*     /docs/{{ $latest }}/:splat
+/docs/current     /docs/next/
+/docs/current/*     /docs/next/:splat
+/docs/v2    /docs/v2.3/
 /docs/v2    /docs/v2.3/
```
Comment (chalin) [inline layouts/index.redirects:7]: Dup'd line

**Review by chalin — APPROVED** [review]

File: layouts/partials/home/hero.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
       <br />
 
       <div class="buttons is-centered">
-        <a class="button is-primary is-inverted is-medium is-radiusless" href="/docs/current/">
+        <a class="button is-primary is-inverted is-medium is-radiusless" href="/docs/latest/">
```
Comment (nate-double-u) [inline layouts/partials/home/hero.html:20]: I'm not happy that this breaks the button on `localhost`, even when using `netlify dev` this doesn't forward correctly (it brings you to http://localhost:8888/docs/), but it does work on the deploy preview/will work on the production build.

File: layouts/index.redirects (hunk 1 above)
Comment (nate-double-u) [reply layouts/index.redirects:7]: thanks, good catch

File: layouts/partials/home/hero.html (hunk 2 above)
Comment (chalin) [reply layouts/partials/home/hero.html:20]: Hmm, it works for me.

Comment (chalin) [reply layouts/partials/home/hero.html:20]: Oh, maybe not ... wait.

Comment (chalin) [reply layouts/partials/home/hero.html:20]: I can confirm that it works for me locally using `netlify dev`

Comment (nate-double-u) [reply layouts/partials/home/hero.html:20]: I verify your confirmation. Thanks!

Comment (nate-double-u) [reply layouts/partials/home/hero.html:20]: No, I take back my verification -- the frontpage button on `localhost:8888` takes me to http://localhost:8888/docs/, not http://localhost:8888/docs/v3.4/

**Comment by nate-double-u** [comment]
Corrected nav:
* https://deploy-preview-161--etcd.netlify.app/docs/v2.3/libraries-and-tools/
  Note, next no longer links to v2.3 and correctly links to the top level of next.

Corrected deprecation warning:
* https://deploy-preview-161--etcd.netlify.app/docs/next

**Review by chalin — APPROVED** [review]

**Pull request merged**